tox.ini

```diff
@@ -119,13 +125,15 @@ commands=
 basepython=python3.5
 deps =
     pytest==3.0.2
+    pytest-xdist==1.15.0
 commands=
     python -m pytest tests/pytest tests/cover/test_testdecorators.py
 
 [testenv:pytest28]
 basepython=python3.5
 deps =
     pytest==2.8.2
+    pytest-xdist==1.15.0
```

Comment:
Hmm. We could probably have a requirements-test.txt and use that. It causes tox some problems (it doesn't notice when the file has changed) but for our use case that's not a big deal given that we trash the tox environment before run.